.github/workflows/ci.yml

```
@@ -32,47 +26,49 @@ jobs:
         python setup.py build_ext --inplace
     
-    - name: Build C++ extensions (Linux only)
-      if: matrix.backend == 'c' && runner.os == 'Linux'
+    - name: Build C++ extensions
+      if: matrix.backend == 'c'
       run: |
-        python setup_cpp.py build_ext --inplace
+        pip install pybind11
+        python setup_extensions/setup_buffer_cpp.py build_ext --inplace || true
+        python setup_extensions/setup_cpp.py build_ext --inplace || true
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)